posts/models.py

```diff
@@ -359,10 +359,11 @@ def filter_for_main_feed(self):
         """
 
         return self.filter(
-            Q(default_project__add_posts_to_main_feed=True)
+            Q(default_project__visibility=Project.Visibility.NORMAL)
             | Exists(
                 Post.projects.through.objects.filter(
-                    post_id=OuterRef("pk"), project__add_posts_to_main_feed=True
+                    post_id=OuterRef("pk"),
+                    project__visibility=Project.Visibility.NORMAL,
```

Comment:
See comment on `projects/models.py`
Just want to make sure accidental setting of visibility on Tags and other non-question project types don't screw this up.